docker/docker-compose.yml

```diff
@@ -17,6 +17,7 @@ services:
         - ../LEAF_Request_Portal:/var/www/html/LEAF_Request_Portal
         - ../libs:/var/www/html/libs
         - ../test:/var/www/html/test
+        - ../:/var/www/html/
```

Comment:
if you need the entire root dir then remove these  
        - ../LEAF_Nexus:/var/www/html/LEAF_Nexus
        - ../LEAF_Request_Portal:/var/www/html/LEAF_Request_Portal
        - ../libs:/var/www/html/libs
        - ../test:/var/www/html/test

Reply:
done